### ISSUE TITLE
Connection between 2 mcu

### ISSUE THREAD

**Issue by pavellzubkov** [jeelabs/esp-link#385]
Can i use esp-link to establish connection between 2 mcu or it impossible? I trying connect 2 esp8266 with esp-link each to other, but uart is silent...

**Comment by yesyesuk** [comment]
I don't think this works currently. esp-link is "only" a TCP server and not a TCP client. For a direct connection you would need a server and a client to establish a connection.

**Comment by uzi18** [comment]
It is possible with use of ELClient library on one side.
